templates/livecoms-template-bestpractices.tex

```diff
@@ -243,12 +243,46 @@ \subsection{Lists}
 \item[Idea] Text
 \end{description}
 
-Some filler text, because empty templates look really poorly. \lipsum[1]
-
-
-\section{Acknowledgments}
-
-Funder and other information can be given here.
+Some filler text, because empty templates look really poor. \lipsum[1]
+
+\section{Author Contributions}
+%%%%%%%%%%%%%%%%
+% Attribute authors into the general categories:
+% * ''wrote sections X and Y''.
+% * ''contributed to all sections''.
+% * ''contributed to section X.''
+% * ''performed and analyzed the experiments in section Y''.
+% See the policies ``Policies on Authorship'' section of https://livecoms.github.io for more information on
```

Comment:
Can we link to it rather than just saying to see it? Or are you worried link will change?

Reply:
Also, seems like we should have an example of what this section might look like; right now the comments you have here are (a) hidden in the TeX, and (b) sound to me like we're expecting this to be very brief and just assign authors into THESE categories, such as "David wrote sections X and Y and performed and analyzed the experiments in section Y", as opposed to "David wrote X, Y, performed and analyzed the experiments in Z, made figures A-Z, and edited the whole manuscript as well as standardizing table formatting" or some such.

I'd probably do it by (a) giving an example, and (b) suggesting some categories like yours (but also including editing, making figures, gathering data/citations, building tables, etc.) that the authors MAY want to use. I'd probably want to avoid saying that they SHOULD use these because they should do whatever best describes their contributions.

Reply:
> Can we link to it rather than just saying to see it? Or are you worried link will change? 

If it's in % comments, then we can't really link, it's plain text.  

Responding to the rest
>  (a) hidden in the TeX,

So I think we do what instructions hidden in comments. That's where I would expect the instructions to use the template.  Whether they are the right instructions, that's something else.  I'm fine with just suggesting a lot of categories, per your comments on the web site.

Reply:
> If it's in % comments, then we can't really link, it's plain text.

I just mean, it's easier for people to get there if you say https://livecomsjournal.github.io/#authorship-and-changes-to-authorship than "policies and authorship section of livecomsjournal.github.io".

> So I think we do what instructions hidden in comments. 

I like instructions hidden in comments. I just mean you might want an example here too.